File: 2-semestre/uri-online-judge/c/1010.c
#include <stdio.h>

int main() {
    int codPeca1, numPeca1, codPeca2, numPeca2;
    double valUniPeca1, valUniPeca2, totValor;
    
    scanf("%d", &codPeca1);
    scanf("%d", &numPeca1);
    scanf("%lf", &valUniPeca1);
    scanf("%d", &codPeca2);
    scanf("%d", &numPeca2);
    scanf("%lf", &valUniPeca2);
    totValor = (numPeca1 * valUniPeca1) + (numPeca2 * valUniPeca2);
    
    printf("VALOR A PAGAR: R$ %.2lf\n", totValor);
    
    return 0;
}

File: 2-semestre/uri-online-judge/javascript/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let primeiro = lines.shift().split(" ");
let segundo = lines.shift().split(" ");

let codPeca1 = parseInt(primeiro[0]);
let numPeca1 = parseInt(primeiro[1]);
let valUniPeca1 = parseFloat(primeiro[2]);

let codPeca2 = parseInt(segundo[0]);
let numPeca2 = parseInt(segundo[1]);
let valUniPeca2 = parseFloat(segundo[2]);

let total = (numPeca1 * valUniPeca1) + (numPeca2 * valUniPeca2);

console.log("VALOR A PAGAR: R$ " + total.toFixed(2));
